.travis.yml

```diff
@@ -217,6 +217,15 @@ matrix:
     - mkdir -p target/kcov
     - RUST_BACKTRACE=1 cargo coverage --verbose
     - bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
+  # Go
+  - language: go
+    os: linux
+    go:
+    - 1.10.x
+    script:
+    - go get -d -t -v ./go/arrow/...
```

Comment:
Can you add a line `- if [ $ARROW_CI_GO_AFFECTED != "1" ]; then exit; fi` here so that the Go build only runs for GO PRs?

Reply:
Sure. Will do when in front of a real keyboard.

Reply:
Rebased and added that